### DISCUSSION TITLE
更新日志 - 22.03.5 - 20230922

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1013] [Announcements]
# 应用
* QuickStart: 修改首页软件源的检测逻辑，以避免将软件源页面错误判定为软件源错误

# 设备
* x86/RK35xx系列机型：移除上一版本中增加的 EROFS 模块，此模块导致内存异常占用

~~x86 固件增加一个测试版本，同样是 22.03.5，但移植了 i915-oot 驱动，理论上可支持最新的 Intel GPU（包括最新的 Meteor Lake/14 代），目前在 N5105 和 N100 机型测试通过，但测试发现 J4125 等老机型会循环重启，有需要的用户可以手动下载安装测试，下载链接： [EFI版](https://fw0.koolcenter.com/iStoreOS/alpha/x86_64_efi/istoreos-22.03.5-2023092117_I915_OOT-x86-64-squashfs-combined-efi.img.gz) ｜ [非EFI版](https://fw0.koolcenter.com/iStoreOS/alpha/x86_64/istoreos-22.03.5-2023092117_I915_OOT-x86-64-squashfs-combined.img.gz)~~ (最新正式版固件已支持，不需要使用测试固件)

**Comment by Ginua8292** [comment]
刷了几个版本！不知道是不是BUG 我的是R4S 4G! 应用过滤插件的 用户列表一直没有，第二天依然没！还有上存特征库文件也是没有！还是我哪里没打开读不到用户列表和上存不了特征库？还是因为我修改了IP就不能用了 比如192.168.100.1改成了192.168.1.1

**Reply by jjm2473** [reply]
你启用oaf没

**Comment by LeungKowk** [comment]
ARS2也有将软件源页面错误判定为软件源错误，请问何时修复？
